### PR TITLE
android/*: add Russian translation

### PR DESCRIPTION
**Pull request by dmorozov001**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ru/android/logcat.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Дамп лог (журнал) системных сообщений, включая трассировку стека при возникновении ошибки и информационные сообщения, регистрируемые приложениями.
+> Дополнительная информация: <https://developer.android.com/studio/command-line/logcat>.
+
+- Показать системные логи:
+
+`logcat`
+
+- Записать системные логи в файл:
+
+`logcat -f {{path/to/file}}`
```
Comment (kbdharun) [inline pages.ru/android/logcat.md:12]:
```suggestion
`logcat -f {{путь/до/файла}}`
```

File: pages.ru/android/pkg.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> Утилита управления пакетами для Termux.
+> Дополнительная информация: <https://wiki.termux.com/wiki/Package_Management>.
+
+- Обновить все установленные пакеты:
+
+`pkg upgrade`
+
+- Установить пакет:
+
+`pkg install {{package}}`
```
Comment (kbdharun) [inline pages.ru/android/pkg.md:12]:
```suggestion
`pkg install {{пакет}}`
```

File: pages.ru/android/pkg.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Удалить пакет:
+
+`pkg uninstall {{package}}`
```
Comment (kbdharun) [inline pages.ru/android/pkg.md:16]:
```suggestion
`pkg uninstall {{пакет}}`
```

File: pages.ru/android/pkg.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Переустановить пакет:
+
+`pkg reinstall {{package}}`
```
Comment (kbdharun) [inline pages.ru/android/pkg.md:20]:
```suggestion
`pkg reinstall {{пакет}}`
```

File: pages.ru/android/pkg.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Поиск пакета:
+
+`pkg search {{package}}`
```
Comment (kbdharun) [inline pages.ru/android/pkg.md:24]:
```suggestion
`pkg search {{пакет}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, Welcome to tldr. I have some suggestions for this page.

Also, translate command arguments inside `{{}}` whenever possible as it will make the placeholders to be easily understood by native readers.

File: pages.ru/android/am.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# am
+
+> Android activity manager.
```
Comment (kbdharun) [inline pages.ru/android/am.md:3]: Translate this line too.

Tip: Batch all my suggestions and make a single commit from `Files changed` tab.

**Comment by kbdharun** [comment]
Thanks for the quick response. Let's see what the other review thinks.

**Comment by dmorozov001** [comment]
Thank you for the suggestions!

File: pages.ru/android/am.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# am
+
+> Менеджер активностей Android.
+> Дополнительная информация: <https://developer.android.com/studio/command-line/adb#am>.
```
Comment (navarroaxel) [inline pages.ru/android/am.md:4]: We use this standard from https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py
```suggestion
> Больше информации: <https://developer.android.com/studio/command-line/adb#am>.
```

**Comment by dmorozov001** [comment]
@EmilySeville7cfg , kindly requesting review :)

File: pages.ru/android/bugreport.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# bugreport
+
+> Показать отчет об ошибках Android.
+> Эту команду можно использовать только через `adb shell`.
+> Дополнительная информация: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/bugreport>.
```
Comment (navarroaxel) [inline pages.ru/android/bugreport.md:5]: Update all pages
```suggestion
> Больше информации: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/bugreport>.
```

**Comment by dmorozov001** [comment]
@navarroaxel , done! Thank you for the suggestion.

File: pages.ru/android/dalvikvm.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> Виртуальная машина Android Java.
+> Больше информации: <https://source.android.com/devices/tech/dalvik>.
+
+- Запустить Java-программу:
+
+`dalvikvm -classpath {{path/to/file.jar}} {{classname}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/dalvikvm.md:8]:
```suggestion
`dalvikvm -classpath {{путь/к/файлу.jar}} {{classname}}`
```

- This change is optional - no corresponding style guide requirement exists.
- It's better to provide placeholder translations when possible.

File: pages.ru/android/dumpsys.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+> Эту команду можно использовать только через `adb shell`.
+> Больше информации: <https://developer.android.com/studio/command-line/dumpsys>.
+
+- Получить диагностическу для всех системных сервисов:
+
+`dumpsys`
+
+- Получить диагностическу для конкретной системного сервисв:
+
+`dumpsys {{service}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/dumpsys.md:13]:
```suggestion
`dumpsys {{сервис}}`
```

The same issue as described above.

File: pages.ru/android/dumpsys.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+- Список всех сервисов доступных через `dumpsys`:
+
+`dumpsys -l`
+
+- Задать специфичные для сервиса аргументы:
+
+`dumpsys {{service}} -h`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/dumpsys.md:21]:
```suggestion
`dumpsys {{сервис}} -h`
```

The same issue as described above.

File: pages.ru/android/dumpsys.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+- Исключить конкретный сервис из диагностики:
+
+`dumpsys --skip {{service}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/dumpsys.md:25]:
```suggestion
`dumpsys --skip {{сервис}}`
```

The same issue as described above.

File: pages.ru/android/dumpsys.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+
+- Задать время ожидания в секундах (по умолчанию 10 сек):
+
+`dumpsys -t {{seconds}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/dumpsys.md:29]:
```suggestion
`dumpsys -t {{секунды}}`
```

The same issue as described above.

File: pages.ru/android/input.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+
+`input text "{{text}}"`
+
+- Отправить одно нажатие на экран на устройство Android:
+
+`input tap {{x_pos}} {{y_pos}}`
+
+- Отправить жест смахивания на устройство Android:
+
+`input swipe {{x_start}} {{y_start}} {{x_end}} {{y_end}} {{duration_in_ms}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/input.md:21]:
```suggestion
`input swipe {{x_начало}} {{y_начало}} {{x_конец}} {{y_конец}} {{продолжительность_в_мс}}`
```

The same issue as described above.

File: pages.ru/android/input.md [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+
+`input keyevent {{event_code}}`
+
+- Отправить текст на устройство Android (`%s` означает пробел):
+
+`input text "{{text}}"`
+
+- Отправить одно нажатие на экран на устройство Android:
+
+`input tap {{x_pos}} {{y_pos}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/input.md:17]:
```suggestion
`input tap {{х_позиция}} {{y_позиция}}`
```

The same issue as described above.

File: pages.ru/android/input.md [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+> Эту команду можно использовать только через `adb shell`.
+> Больше информации: <https://developer.android.com/reference/android/view/KeyEvent.html#constants_1>.
+
+- Отправить код события для одного символа на устройство Android:
+
+`input keyevent {{event_code}}`
+
+- Отправить текст на устройство Android (`%s` означает пробел):
+
+`input text "{{text}}"`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/input.md:13]:
```suggestion
`input text "{{текст}}"`
```

The same issue as described above.

File: pages.ru/android/input.md [hunk 17]
```diff
@@ -0,0 +1,25 @@
+# input
+
+> Отправить коды событий или жесты сенсорного экрана на устройство Android.
+> Эту команду можно использовать только через `adb shell`.
+> Больше информации: <https://developer.android.com/reference/android/view/KeyEvent.html#constants_1>.
+
+- Отправить код события для одного символа на устройство Android:
+
+`input keyevent {{event_code}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/input.md:9]:
```suggestion
`input keyevent {{код_события}}`
```

The same issue as described above.

File: pages.ru/android/input.md [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+
+`input tap {{x_pos}} {{y_pos}}`
+
+- Отправить жест смахивания на устройство Android:
+
+`input swipe {{x_start}} {{y_start}} {{x_end}} {{y_end}} {{duration_in_ms}}`
+
+- Отправить длинное нажатие на экран на устройство Android с помощью жеста смахивания:
+
+`input swipe {{x_pos}} {{y_pos}} {{x_pos}} {{y_pos}} {{duration_in_ms}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/input.md:25]:
```suggestion
`input swipe {{x_position}} {{y_position}} {{x_position}} {{y_pos}} {{продолжительность_в_мс}}`
```

The same issue as described above.

File: pages.ru/android/logcat.md [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+
+`logcat`
+
+- Записать системные логи в файл:
+
+`logcat -f {{путь/до/файла}}`
+
+- Показать строки, соответствующие регулярному выражению:
+
+`logcat --regex {{regular_expression}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/logcat.md:16]:
```suggestion
`logcat --regex {{регулярное_выражение}}`
```

The same issue as described above.

File: pages.ru/android/pm.md [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+
+`pm list packages -s`
+
+- Показать список всех установленных сторонних приложений:
+
+`pm list packages -3`
+
+- Показать список приложений по ключевым словам:
+
+`pm list packages {{keywords}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/pm.md:20]:
```suggestion
`pm list packages {{ключевые_слова}}`
```

The same issue as described above.

File: pages.ru/android/screencap.md [hunk 21]
```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Сделать снимок экрана мобильного дисплея.
+> Эту команду можно использовать только через `adb shell`.
+> Больше информации: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Сделать снимок экрана:
+
+`screencap {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/screencap.md:9]:
```suggestion
`screencap {{путь/к/файлу}}`
```

The same issue as described above.

File: pages.ru/android/pm.md [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+
+- Показать путь к APK определенного приложения:
+
+`pm path {{app}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/android/pm.md:24]:
```suggestion
`pm path {{приложение}}`
```

The same issue as described above.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found.
- Thanks for a massive contribution! ❤️ 🚀

**Review by kbdharun — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**